static/src/javascripts/projects/commercial/modules/messenger/background.js

```diff
@@ -194,7 +232,18 @@ const setBackground = (specs: AdSpec, adSlot: Node): Promise<any> => {
         .then(({ backgroundParent, background }) =>
             updateStyles(backgroundParent, background)
         )
-        .then(({ backgroundParent }) => observer.observe(backgroundParent));
+        .then(({ backgroundParent, background }) => {
+            if (specs.scrollType === 'interscroller') {
+                renderStickyAdLabel(backgroundParent).then();
+                renderStickyScrollForMoreLabel(backgroundParent).then();
+
+                addEventListener(window, 'scroll', () =>
+                    onInterscrollerScroll(backgroundParent, background)
+                );
```

Comment:
When listening to `scroll` events we should use a "passive event listener" to improve scroll performance (https://developers.google.com/web/tools/lighthouse/audits/passive-event-listeners).

To do this you can pass a fourth "options" argument to `addEventListener`:

```js
addEventListener(window, 'scroll', () => onInterscrollerScroll(backgroundParent, background), {passive: true});
```

Reply:
Thanks, added it now